File: backend/src/main/java/com/ndoruhirwe/smartlogistics/dto/response/PurchaseOrderItemResponse.java
package com.ndoruhirwe.smartlogistics.dto.response;

import java.math.BigDecimal;
import java.util.UUID;

public record PurchaseOrderItemResponse(

        UUID id,

        UUID productId,

        String productSku,

        String productName,

        BigDecimal orderedQuantity,

        BigDecimal receivedQuantity,

        BigDecimal remainingQuantity,

        BigDecimal unitPrice,

        BigDecimal lineTotal
) {
}

File: frontend/src/app/features/procurement/models/purchase-order-item-response.ts
export interface PurchaseOrderItemResponse {
  id: string;

  productId: string;
  productSku: string;
  productName: string;

  orderedQuantity: number;
  receivedQuantity: number;
  remainingQuantity: number;

  unitPrice: number;
  lineTotal: number;
}
